services/scan/src/interpreter.test.ts

```diff
@@ -12,15 +12,19 @@ import {
   unsafeParse,
 } from '@votingworks/types';
 import { throwIllegalValue } from '@votingworks/utils';
-import { readFile } from 'fs-extra';
+import { readFile, emptyDirSync } from 'fs-extra';
 import { join } from 'path';
 import * as choctaw2020Fixtures from '../test/fixtures/2020-choctaw';
 import * as stateOfHamiltonFixtures from '../test/fixtures/state-of-hamilton';
+import * as msDemoFixtures from '../test/fixtures/election-b0260b4e-mississippi-demo';
 import { Interpreter, sheetRequiresAdjudication } from './interpreter';
+import { createInterpreter } from './simple_interpreter';
 import { pdfToImages } from './util/pdf_to_images';
 import { detectQrcodeInFilePath } from './workers/qrcode';
 
 const sampleBallotImagesPath = join(__dirname, '..', 'sample-ballot-images/');
+const interpreterOutputPath = join(__dirname, '..', 'test-output-dir/');
```

Comment:
Any reason not to just create a temp dir?

Reply:
no good reason! Probably a better solution.